### ISSUE TITLE
What to do with an installation we find that isn't on the map?

### ISSUE THREAD

**Issue by IlariaBelvedere** [IQSS/dataverse-installations#145]
Hello,

checking on re3data I found a few Dataverse installations that are not listed on the map (as @pdurbin said, being on the map isn't mandatory and some of them could be in their demo phase). 
What do we do with them? Could it be interesting to add a new sheet on the file about installations but specifying they are not on the map (for several reasons which maybe it is not right to explicitate)?
Give this matter an hand, please :)

**Comment by shlake** [comment]
@IlariaBelvedere 
Not sure about all of the RE3Data Dataverse entries you found, but this one is actually a sub dataverse (collection) on Harvard’s 
https://www.re3data.org/repository/r3d100010713

This one is as well:
https://www.re3data.org/repository/r3d100010949

They are not separate installations.

**Comment by jggautier** [comment]
(The [previous comment](https://github.com/IQSS/dataverse-installations/issues/145#issuecomment-1153998445) was somehow duplicated so I delete the second one.)

Shouldn't we try to contact each installation to see if they'd like to be added?

**Comment by pdurbin** [comment]
@IlariaBelvedere yes! Please feel free to email folks. Do you have Marion Wittenberg's address? She recently posted about a new installation here: https://groups.google.com/g/dataverse-community/c/7cUyLesUk-Q/m/7XM3UcirHAAJ

**Comment by IlariaBelvedere** [comment]
I will try my best then :)
No, I haven't got her email address, do you?
Thanks,

Ilaria

Il 13 lug 2022 20:23, Philip Durbin ***@***.***> ha scritto:

@IlariaBelvedere<https://github.com/IlariaBelvedere> yes! Please feel free to email folks. Do you have Marion Wittenberg's address? She recently posted about a new installation here: https://groups.google.com/g/dataverse-community/c/7cUyLesUk-Q/m/7XM3UcirHAAJ

**Comment by pdurbin** [comment]
@IlariaBelvedere yep, I just emailed her and cc'd you. Thanks!

**Comment by pdurbin** [comment]
@IlariaBelvedere your outreach efforts are working! Thanks! Check out this issue:

- #160

**Comment by jggautier** [comment]
I mentioned contacting these repositories and meant that this could be done instead of keeping an internal list of installations that aren't on the map. I'm curious why we'd want to keep a list of Dataverse installations that aren't on the map? Here are some reasons I can think of:

- So that we can consider these installations when we review the different types of Dataverse installations, such as for the "installation archetype" work that was done a while back, which can help inform what kinds of use-cases the Dataverse community supports
- So that we keep a record of when the installation was last contacted and who contacted them, to avoid contacting them too often but to also know when to ask again if they'd like to be added to the map

Are there other reasons for keeping a list of Dataverse installations that aren't on the map?

**Comment by pdurbin** [comment]
Another one! Thanks @IlariaBelvedere for reaching out to them! And thanks @shlake for adding it. 🎉 

- #164

@jggautier I think of it as a sales funnel, I guess. Even though Dataverse is free as in beer. Or maybe conversion is the right term? Conversion rates? You want people to buy something. Or in our case, you want them to opt in to being on the map, to join our community. So it makes sense to check in with installations. How's it going? Do you need anything? Did we tell you you can put your installation on a map? That sort of thing. I guess this is basically your second bullet but I don't think we actually track when we last contacted anyone.

**Comment by jggautier** [comment]
I forgot that on [the spreadsheet](https://docs.google.com/spreadsheets/d/1bfsw7gnHlHerLXuk7YprUT68liHfcaMxs1rFciA-mEo/edit#gid=0) you also added a "skip" column to record info about installations that didn't want to be on the map right away. So maybe we could add installation info to the spreadsheet and in the skip column indicate with a "yes" or "true" or something else that the installation shouldn't be added to the map. The code seems to be checking for any value in that column.

If one of the goals is to avoid multiple people contacting installations too often, so they're not bothered, I think some indication of the last time they were contacted would be helpful. Maybe a last_contacted_date column could be added for that info, and any date entered can be removed once the installation is added to the map.

I like the analogy of conversation rates. But I think we could be better about how certain terminology is used wherever these metrics are publicized. For example, for groups running what they would call "repositories", we often call those "collections" or "Dataverse collections" within "installations" or "instances". So here it might be helpful to consistently use these terms.
